File: frontend/lib/models/onec_sync_status_dto.dart
class OneCSyncStatusDto {
  final String? startedAt;
  final String? finishedAt;
  final bool success;
  final String? trigger;
  final String? message;
  final int imported;
  final int updated;
  final int skipped;

  const OneCSyncStatusDto({
    this.startedAt,
    this.finishedAt,
    required this.success,
    this.trigger,
    this.message,
    required this.imported,
    required this.updated,
    required this.skipped,
  });

  factory OneCSyncStatusDto.fromJson(Map<String, dynamic> json) {
    return OneCSyncStatusDto(
      startedAt: json['startedAt'] as String?,
      finishedAt: json['finishedAt'] as String?,
      success: json['success'] == true,
      trigger: json['trigger'] as String?,
      message: json['message'] as String?,
      imported: (json['imported'] as num?)?.toInt() ?? 0,
      updated: (json['updated'] as num?)?.toInt() ?? 0,
      skipped: (json['skipped'] as num?)?.toInt() ?? 0,
    );
  }
}

File: backend/src/main/java/ru/bowling/bowlingapp/integration/onec/dto/OneCSyncStatusDto.java
package ru.bowling.bowlingapp.integration.onec.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OneCSyncStatusDto {
    private LocalDateTime startedAt;
    private LocalDateTime finishedAt;
    private Boolean success;
    private String trigger;
    private String message;
    private Integer imported;
    private Integer updated;
    private Integer skipped;
}
